daemon/config/config.go

```diff
@@ -54,11 +54,10 @@ const (
 	DefaultContainersNamespace = "moby"
 	// DefaultPluginNamespace is the name of the default containerd namespace used for plugins.
 	DefaultPluginNamespace = "plugins.moby"
-	// defaultMinAPIVersion is the minimum API version supported by the API.
+	// minAPIVersion is the minimum API version supported by the API.
 	// This version can be overridden through the "DOCKER_MIN_API_VERSION"
-	// environment variable. The minimum allowed version is determined
-	// by [minAPIVersion].
-	defaultMinAPIVersion = "1.24"
+	// environment variable.
+	minAPIVersion = "1.24"
```

Comment:
Hmm. actually, maybe I should add back a const in the `api` package to prevent code there from pretending it supports older API versions.